business/products.md

```diff
@@ -17,31 +17,46 @@ They vary by:
 and other features, as well as underlying architecture.
 
 
-The advantage of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
+The alternative of using a machine translation provider is that the service is already working and can be integrated immediately. The alternative is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
```

Comment:
seems like a mistake to change this to "alternative"

Better "An advantage of" and "An alternative to"